MPF.Processors/Redumper.cs

```diff
@@ -673,6 +675,22 @@ private static bool IsManufacturerEmpty(string inputFilename)
             }
         }
 
+        #endregion
+        
+        #region Private Enums
+
+        /// <summary>
+        /// Private enum for SecuROM scheme type
+        /// </summary>
+        internal enum SecuROMScheme
```

Comment:
This will fly for now, but it really should live in an `Enums.cs` file in the Processors project later.